server/app/rest_api/rest.py

```diff
@@ -248,7 +246,7 @@ class AnnotationsVarAPI(Resource):
     def get(self):
         fields = request.args.getlist("annotation-name", None)
         try:
-            annotation_response = current_app.data.annotation(current_app.data.data, "var", fields)
+            annotation_response = current_app.data.annotation(None, "var", fields)
```

Comment:
You're passing in a `None` when your docblock calls this a dict.  Expected?

Reply:
That is a valid approach.  It's also valid to just update the documentation to say that `None` == `{}`.